lms/djangoapps/dashboard/tests/test_sysadmin.py

```diff
@@ -290,6 +290,7 @@ def test_gitlog_no_logs(self):
 
         self._rm_edx4edx()
 
+    @unittest.skipIf(settings.SKIP_EDRAAK_UNWANTED_TESTS, 'Could not fix it on CircleCI.')
```

Comment:
this particular test is very weird, in our fork, it [fails one run](https://circleci.com/gh/eduNEXT/edunext-platform/67), and then [passes the next](https://circleci.com/gh/eduNEXT/edunext-platform/68)